test/show.jl

```diff
@@ -35,4 +38,7 @@ module TestShow
     show(io, A)
     A = DataFrames.RepeatedVector([1, 2, 3], 1, 5)
     show(io, A)
+
+    render(Juno.Editor(), df)
```

Comment:
I don't really see why. The important thing on the DataFrames side is that it's using Juno.jl's API correctly, and calling the function tests that. All of the actual functionality and logic that could regress is on the Juno side and can be tested there.